kit/foundation/meshstack/guides/guide_meshstack_hub.md

```diff
@@ -0,0 +1,57 @@
+# **Demo Story: Importing Ready-to-Use Infrastructure with meshStack Hub**
+
+This guide shows you how you can accelerate your cloud governance with pre-built building blocks from meshStack Hub
+
+## Motivation
+
+Likvid Bank is a growing financial services provider embracing cloud infrastructure to scale digital services.
```

Comment:
I am wondering how much "story setting" we need in each demo story. Might be worth to put these general things into "setting the scene" to avoid the misalignment

Reply:
I made it smaller with less of a story more with the direct motivation